openapi/kinto/types.bal

```diff
@@ -15,11 +15,11 @@
 // under the License.
 
 public type BatchResponseBodySchema record {
-    record  { record {} body?; record {} headers?; string path; int status;} [] responses;
+    record {record {} body?; record {} headers?; string path; int status;}[] responses;
 };
 
 public type Schema record {
-    record {}[] data?;
+    record {record {} 'collection\:schema?; record {} 'group\:schema?; record {} 'record\:schema?;}[] data?;
```

Comment:
Please check on the names of these records

Reply:
This package builds without errors. Is there any specific reason to check the name?

Reply:
we cannot format the names since we are from the API. To allow fields with special characters to compile successfully we escape special characters.